source/stylesheets/refills/_footer-2.scss

```diff
@@ -71,19 +49,22 @@
   } 
 
   ul {
-    margin-bottom: 1em;
+    line-height: 1.5em;
+    margin: 0 0 $small-spacing 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $small-spacing` instead of `0 0 $small-spacing 0`